terraform/variables.tf

```diff
@@ -103,6 +103,10 @@ variable "hostnames" {
   type    = list(any)
 }
 
+variable "qualified_teachers_api_storage_account_name" {
```

Comment:
Haven't added this to the .tfvars files yet as not sure what name to use. Also there's already a var defined called storage_account_name (e.g. storage_account_name = "s165d01dqtapitfstatedv") that doesn't appear to be used anywhere

Reply:
`storage_account_name` is used by this workflow action https://github.com/DFE-Digital/qualified-teachers-api/blob/main/.github/workflows/actions/deploy-environment/action.yml#L40

For naming, there's a convention that's enforced so something like `s165d01dqtapiappdv` should do it (the same as `storage_account_name` with `tfstate` swapped for `app`).